calendar.php

```diff
@@ -140,11 +140,14 @@
                                 if (isset($events[$e])) {
                                     $dayEvents = $events[$e];
                                     foreach ($dayEvents as $info) {
-                                        if($info["completed"] == "no"){
+                                        if($info["completed"] == "no"){ //Uncompleted events
                                             $eventsStr .= '<a class="calendar-event" style="background-color:#1a7024" href="event.php?id=' . $info['id'] . '">' . $info['name'] .  '</a>';
-                                        } else {
-                                            $eventsStr .= '<a class="calendar-event" href="event.php?id=' . $info['id'] . '">' . $info['abbrevName'] .  '</a>';
+                                        } else { //Completed Events
+                                            $eventsStr .= '<a class="calendar-event" href="event.php?id=' . $info['id'] . '">' . $info['name'] .  '</a>';
                                         }
+                                        //if($info['restricted'] == 1) {
```

Comment:
What was this test code for? If it is related to restricted events (something we will have) I wonder if we could repurpose it somehow. It might not be useful to us though.

Reply:
This is for modifying the appearance of each Event block depending on if the event is restricted. The next code would be

```
if($info['restricted_signup'] == 1) { //Restricted Event
    $backgroundCol = '#c73d06'; // Sets background color to a red to show the restriction
} else {
    $backgroundCol = '#1a7024'; //Otherwise keeps background as green
}
```

Reply:
Updated code to reflect comments/new feature